### ISSUE TITLE
Drops 9?

### ISSUE THREAD

**Issue by romulasry** [pantheon-systems/drops-8#389]
Where is it?

**Comment by greg-1-anderson** [comment]
Pantheon only supports Drupal 9 on Composer-managed sites. To convert a drops-8 site to an Integrated Composer upstream, please see https://pantheon.io/docs/guides/composer-convert

**Comment by romulasry** [comment]
Okay but I installed the site via Drupal 9 composer and get this error when I do the finishing install.php:

https://www.drupal.org/project/drupal/issues/3255318

**Comment by namespacebrian** [comment]
It looks like integrated composer is not running for your site. Try adding a new line to your `pantheon.yml`:

`build_step: true`

**Comment by romulasry** [comment]
Same error.

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by namespacebrian** [comment]
> Same error. 

Yeah I deleted my message because I saw that it was not actually the problem.. but you got the email notification anyway :)

**Comment by romulasry** [comment]
$ composer up -W
Loading composer repositories with package information
Updating dependencies
Nothing to modify in lock file
Installing dependencies from lock file (including require-dev)
Nothing to install, update or remove
Package doctrine/reflection is abandoned, you should avoid using it. Use roave/better-reflection instead.
Package webmozart/path-util is abandoned, you should avoid using it. Use symfony/filesystem instead.
Generating autoload files
82 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
Scaffolding files for pantheon-systems/drupal-integrations:
  - Skip [web-root]/sites/default/settings.php because it already exists and overwrite is false.
[composer.zip](https://github.com/pantheon-systems/drops-8/files/7749777/composer.zip)

**Comment by romulasry** [comment]
> ```
> Package doctrine/reflection is abandoned, you should avoid using it. Use roave/better-reflection instead.
https://www.drupal.org/project/drupal/issues/3180351
> Package webmozart/path-util is abandoned, you should avoid using it. Use symfony/filesystem instead.
> ```
https://gorannikolovski.com/blog/upgrading-drupal-9-and-composer-2
"Issue #5 - webmozart/path-util

Package "webmozart/path-util" is abandoned, you should avoid using it. Use "symfony/filesystem" instead.

The package is abandoned because it's been merged into the Symfony Filesystem package, so it's not maintained anymore.

Drush requires it, so there's nothing you can do to fix this warning. Just ignore it, and wait for Drush maintainers to fix it.

**Comment by romulasry** [comment]
The 1st is a drupal issue and will be likely fixed by Drupal 10.

The 2nd is a Drush bug: https://github.com/drush-ops/drush/issues/4935

**Comment by greg-1-anderson** [comment]
I believe the issues described here have been fixed. Leaving open because I feel it is useful to have this issue readily visible in the queue. I will also add:

- drops-8 is deprecated. Please upgrade to Drupal (Composer Managed) as suggested above.
- It is possible (but not recommended or supported) to run Drupal 9 on drops-8, but honestly it's probably less effort to upgrade to Composer.
- It is not possible to run Drupal 10 on drops-8. You must upgrade to use Composer. Do not attempt to get Drupal 10 to work on drops-8; it will fail, and you will waste a lot of time.
